src/app/components/ctaButton/ctaButton.tsx

```diff
@@ -0,0 +1,15 @@
+import Link from "next/link";
+import styles from "./ctaButton.module.css";
+
+interface CtaButtonProps {
+  href: string;
+  buttonText: string;
+}
+
+export default function CtaButton({ href, buttonText }: CtaButtonProps) {
```

Comment:
nit: We could potentially give this component a more generic name so that it can be used for various functions in the future. As is, the Figma file only shows primary buttons, but I assume that we may have different variants in the future (non-CTA) and scoping buttons to scenarios may be overkill. 

Additionally, to make it more reusable, consider simply presenting the buttonText and optional (maybe, ask Danny) icon and allowing the consumer of this component to handle what happens on click (notice the onClick callback prop below). 

Perhaps something like this?

```tsx
export type ButtonVariant = "primary"; // To be extended later on when design calls for additional variants

export interface ButtonProps {
  buttonText: string;
  variant?: ButtonVariant;
  onClick: () => void;
  showIcon?: boolean;
}

export default function Button({
  buttonText,
  onClick,
  variant = "primary",
  showIcon,
}: ButtonProps) {
  const buttonClass = variant === "primary" ? styles["button--primary"] : "";

  return (
    <button className={`${styles.button} ${buttonClass}`} onClick={onClick}>
      {buttonText}
      {showIcon && <span className={styles.button__icon}>-></span>}
    </button>
  );
}
```

Leaving this feedback as a nit because I can see the argument of creating another ticket to refactor this to a more generic button component. :)

Reply:
This is great feedback, i'll use implement this!